src/scss/components/pages.scss

```diff
@@ -104,7 +109,7 @@
 
 // Page features & quicklinks, used on homepage
 .page-showcase {
-  background-color: lighten($gray-200, 0.5);
+  background-color: lighten($gray-200, 50%);
```

Comment:
this comes out as white - might need to tweak this, according to designs

Reply:
Done!